teuthology/openstack/__init__.py

```diff
@@ -830,13 +830,21 @@ def run_suite(self):
         running the teuthology cluster.
         """
         original_argv = self.argv[:]
-        argv = []
+        argv = ['--ceph', self.args.ceph,
+                '--ceph-repo', self.args.ceph_repo,
+                '--suite-repo', self.args.suite_repo,
+                '--suite-branch', self.args.suite_branch,
+                '--teuthology-branch', self.args.teuthology_branch]
```

Comment:
```$ (virtualenv) smithfarm@localhost:~/src/ceph/SUSE/teuthology/scripts> grep getenv openstack.py
        default=os.getenv('TEUTH_CEPH_BRANCH', 'master'),
        default=os.getenv('TEUTH_SUITE_BRANCH', 'master'),
        default=os.getenv('TEUTH_CEPH_REPO', 'https://github.com/ceph/ceph'),
        default=os.getenv('TEUTH_SUITE_REPO', 'https://github.com/ceph/ceph'),
        default=os.getenv('TEUTH_REPO', 'https://github.com/ceph/teuthology'),
        default=os.getenv('TEUTH_BRANCH', 'master'),
```

It would seem that `--teuthology-git-url` is missing here?

Reply:
@smithfarm yeah, this is weird, but teuthology-suite does not support --teuthology-git-url or --teuthology-repo yet, it is a topic for separate patch with corresponding testing.
The `--teuthology-git-url` is used with teuthology-openstack only though.